app/views/admin/settings/show.html.erb

```diff
@@ -1038,6 +1038,29 @@
           </div>
         <% end %>
 
+        <%= form_for(Settings::SMTP.new, url: admin_settings_smtp_settings_path) do |f| %>
```

Comment:
I think it may be time to break these out into partials. Do you want to start with this form to set a good precedent?